test/model/src/include/model/kv_table_item.h

```diff
@@ -39,6 +39,8 @@
 
 namespace model {
 
+class kv_table;
```

Comment:
Do you need this forward declaration?

Reply:
That's right—I added it because I needed, and then I ended up deleting the code that needed it... Thanks for catching that.